code/game/machinery/engi_points_vendor.dm

```diff
@@ -0,0 +1,183 @@
+
+/obj/machinery/engi_points_manager
+	name = "Engineering Points Manager"
+	desc = "Who's a good boy?"
+	icon = 'icons/obj/machines/engi_points.dmi'
+	icon_state = "store"
+	verb_say = "states"
+	density = 1
+	anchored = 1
+	var/obj/item/device/radio/radio
+	var/GBP = 0
+	var/GBPearned = 0
+	var/power_export_bonus = 0
+	var/air_alarm_bonus = 0
+	var/power_alarm_bonus = 0
+	var/fire_alarm_bonus = 0
+	var/alarm_rating = ""
+	var/prior_bonus = 2500
+	var/total_bonus = 0
+	var/GBP_alarm_cooldown = 4500
+	var/list/prize_list = list(
+		new /datum/GBP_equipment("Tendie",				/obj/item/weapon/reagent_containers/food/snacks/nugget,				50,		1),
+		new /datum/GBP_equipment("Cigar",				/obj/item/clothing/mask/cigarette/cigar/havana,						50,		1),
+		new /datum/GBP_equipment("Soap",				/obj/item/weapon/soap/nanotrasen,									100,	1),
+		new /datum/GBP_equipment("Fulton Beacon",		/obj/item/fulton_core,												50,		1),
+		new /datum/GBP_equipment("Fulton Pack",			/obj/item/weapon/extraction_pack,									200,	1),
+		new /datum/GBP_equipment("Space Cash",			/obj/item/stack/spacecash/c1000,									250,	1),
+		new /datum/GBP_equipment("Insulated Gloves",				/obj/item/clothing/gloves/color/yellow,					400,	1),
+		new /datum/GBP_equipment("50 metal sheets",			/obj/item/stack/sheet/metal/fifty,								500,	1),
+		new /datum/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/glass/fifty,								500,	1),
+		new /datum/GBP_equipment("50 cardboard sheets",			/obj/item/stack/sheet/cardboard/fifty,						500,	1),
+		new /datum/GBP_equipment("Hardsuit x3",			/obj/item/clothing/suit/space/hardsuit,								750,	3),
+		new /datum/GBP_equipment("Jetpack Upgrade x5",		/obj/item/weapon/tank/jetpack/suit,								1000,	5),
+		new /datum/GBP_equipment("Powertools x3",			/obj/item/weapon/storage/belt/utility/chief/full,				2000,	3),
+		new /datum/GBP_equipment("Advanced Magboot x3",			/obj/item/clothing/shoes/magboots/advance,					3000,	3),
+		new /datum/GBP_equipment("Reflector Box x3",			/obj/structure/reflector/box,								3500,	3),
+		new /datum/GBP_equipment("Radiation Collector x3",			/obj/machinery/power/rad_collector,						4000,	3),
+		new /datum/GBP_equipment("Ranged RCD x3",			/obj/item/weapon/rcd/arcd,										6000,	3),
+		new /datum/GBP_equipment("ERT Hardsuit x5",		/obj/item/clothing/suit/space/hardsuit/ert/engi,					7000,	5),

```

Comment:
So as we've said, with perhaps the exception of the repair bike (I've not seen it yet but it seems alright), you need to remove the "placeholder" rewards here down as they're absurd. Even as an admin-spawn machine I've repeatedly seen these all getting var edited out when spawned; If you ever intend for this to be accessible outside of admins the removal of these is beyond question.